test/test_regression.py

```diff
@@ -72,7 +73,7 @@ def teardown_module():
 def get_script_module(script_name):
     """Load a script module"""
     return read_json(os.path.join(retriever_root_dir, "scripts", script_name))
-
+        
```

Comment:
This line should not have a diff.  But it does because there is some created space.

Reply:
I think it is because of the added name of the test.

Reply:
I think this is pasted code. It always happens if you  paste on github.
if you are using command line. after changing the file. Before you do a git add <file>, Do a git diff to see the changes.